Project_source/app/src/main/java/android/clicker/school_live_simulator/Song.kt

```diff
@@ -0,0 +1,9 @@
+package android.clicker.school_live_simulator
+
+enum class Song(val happiness: Int) {
```

Comment:
Enum field should be named as salary, not happiness.

Reply:
Fixed